### ISSUE TITLE
Alternative syntax

### ISSUE THREAD

**Issue by vladim1** [squizlabs/PHP_CodeSniffer#518]
Hi. I have the following code:
```php
<?php foreach ($files as $file): ?>
    <tr>
        <td><?= date('Y-m-d H:i:s', $file['date']) ?></td>
        <td><?= $file['name']) ?></td>
    </tr>
<?php endforeach; ?>
```
PHP_CodeSniffer shows erros(or warnings):
```
phpcs: [x] Expected 1 newline after opening brace;
phpcs: [x] Closing brace must be on a line by itself
```
As I can see the correct syntax should look like this one:
```php
<?php
foreach($files as $file)
?>
<tr>
...
```
But it basically wastes anternative syntax, because the folowing code looks ugly.

I have read PSR1 and PSR2 on php-fig and I couldn't find the requirements specified in the errors. Have I missed something? Or where are the requirements about the new line? 
PS: I have read related topics like this one #305 about alternative syntax and code sniffer.

**Comment by gsherwood** [comment]
PSR1 and 2 were not written with alternate syntax in mind. I suggest you ignore the errors, or don't check template files with those coding standards.
